install_config/cluster_metrics.adoc

```diff
@@ -112,11 +112,11 @@ API
 ----
 
 . Before it can deploy components, the *metrics-deployer* service account must
-also be granted the `edit` permission for the *openshift-infra* project:
+also be granted the `edit` permission for the *openshift-metrics* project:
```

Comment:
This is exactly the sort of thing we're worried about.  This just granted root access to the cluster to the deployer pod installing metrics.  build controller sa -> privileged pods -> pod creation -> docker socket access -> node secrets on disk.

That permission can be revoked later on (its not today), but there's no real reason for it need that access.